openedx/features/caliper_tracking/transformers/navigation_transformers.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+def edx_ui_lms_link_clicked(current_event, caliper_event):
+	caliper_event["type"] = "NavigationEvent"
+	caliper_event["action"] = "NavigateTo"
+	caliper_event["actor"]["type"] = "Person"
+
+	current_event_details = json.loads(current_event["event"])
```

Comment:
Why doing this?

Reply:
I believe we are getting json dump for this event. Is this correct ? @danialmalik

Reply:
Yes, that's right.